build/css.mk

```diff
@@ -12,12 +12,14 @@ dist/css/parts/%: src/css/parts/%
 		build/install.sh 644 $< $@
 
 # CSS rules
-dist/css/min/black-highlighter.min.css: src/css/black-highlighter.css $(BUILD_SOURCES) $(CSS_SOURCES) node_modules
+dist/css/min/black-highlighter.min.css: src/css/black-highlighter.css $(CSS_SOURCES) node_modules
 		build/install.sh 644 $< dist/css/black-highlighter-imports.css
 		bun postcss dist/css/black-highlighter-imports.css --config build --env development -o dist/css/black-highlighter.css
 		bun postcss dist/css/black-highlighter-imports.css --config build --env production -o $@
+		touch $@
 
-dist/css/min/normalize.min.css: src/css/normalize.css $(BUILD_SOURCES) node_modules
+dist/css/min/normalize.min.css: src/css/normalize.css node_modules
 		build/install.sh 644 $< dist/css/normalize.css
 		bun postcss dist/css/normalize.css --config build --env development -r
 		bun postcss dist/css/normalize.css --config build --env production -o $@
+		touch $@
```

Comment:
Shouldn't this be unnecessary, since it writes to the output file each time? Or does `bun` not do anything if it's already up-to-date, leaving the timestamp as-is?

Reply:
It seems bun does not do anything if it is already up-to-date. Without the `touch $@` those commands seem to get run every time.